testsuite/cluster-test/src/tx_emitter.rs

```diff
@@ -377,6 +397,8 @@ impl SubmissionWorker {
                 let now = Instant::now();
                 if wait_util > now {
                     time::delay_for(wait_util - now).await;
+                    // we do not include wait_time as latency
+                    tx_offset_time += (wait_util - now).as_millis() as u64;
```

Comment:
Should we not make this change for now? (since we see that it does not really fix the problem)